src/main/java/com/fasterxml/jackson/datatype/jsr310/deser/LocalDateDeserializer.java

```diff
@@ -75,6 +76,11 @@ public LocalDate deserialize(JsonParser parser, DeserializationContext context)
                 if(string.length() == 0) {
                     return null;
                 }
+
+                if(string.contains("T")) {
+                    return LocalDate.parse(string, DateTimeFormatter.ISO_LOCAL_DATE_TIME);
```

Comment:
Fair point -- I will merge this first, however, just to get things moving (since I was out for a bit and am behind).
If anyone wants to do PR, I'll quickly merge it.